app/lib/clients/bambooHR_client.rb

```diff
@@ -0,0 +1,36 @@
+def find_employees
+  client = Bamboozled.client(subdomain: AppEnv['BAMBOOHR_SUBDOMAIN'], api_key: AppEnv['BAMBOOHR_API_KEY'])
+
+  day_before_yesterday = (Time.zone.yesterday - 1).to_formatted_s(:iso8601)
+  yesterday = Time.zone.yesterday.to_formatted_s(:iso8601)
+  today = Time.zone.now.to_formatted_s(:iso8601)
+  tomorrow = Time.zone.tomorrow.to_formatted_s(:iso8601)
+  day_afer_tomorrow = (Time.zone.tomorrow + 1).to_formatted_s(:iso8601)
+
```

Comment:
Is the information about birthdays, anniversaries & so on displayed only for those days or can you also access for the future/past days?

Reply:
You can access future/past days by changing the date fields (ie today, tomorrow...) I do not believe there is a way to search for the birthday and anniversaries by inputing the date that you want to match on using the API (for example something like this `client.employee.all("hireDate,05-17") ` wouldn't work) but you can do that for the time off field like this `client.time_off.whos_out(Time.now, "2014-12-31")`